homeassistant/components/mqtt/abbreviations.py

```diff
@@ -86,6 +86,7 @@
     "json_attr": "json_attributes",
     "json_attr_t": "json_attributes_topic",
     "json_attr_tpl": "json_attributes_template",
+    "lrprt": "last_report",
```

Comment:
We should not name the option `last_report` IMHO, the new flag disables the suppression of state writes when nothing has changed.

Reply:
renamed to "en_stat_wr_suppr": "enable_state_write_suppression",